site/content/docs/main/release-instructions.md

```diff
@@ -14,6 +14,32 @@ This page covers the steps to perform when releasing a new version of Velero.
 - Pre-releases: Any release leading up to a GA. Example: 1.4.0-beta.1, 1.5.0-rc.1
 - RC releases: Release Candidate, contains everything that is supposed to ship with the GA release. This is still a pre-release.
 
+# Velero Release Requirements
```

Comment:
This header needs to be done with "##" and the headers below this one need to be done with "###". the TOC is not picking up this header.

Reply:
Thanks